src/lib/components/WorkstreamDetail.svelte

```diff
@@ -10,14 +10,41 @@
   import ApplyModal from '$components/ApplyModal.svelte';
   import ApplicationModal from '$components/ApplicationModal.svelte';
   import Apply from 'radicle-design-system/icons/Ledger.svelte';
+  import Cross from 'radicle-design-system/icons/Cross.svelte';
   import ThumbsDown from 'radicle-design-system/icons/ThumbsDown.svelte';
   import Button from 'radicle-design-system/Button.svelte';
+  import Tooltip from 'radicle-design-system/Tooltip.svelte';
   import Markdown from 'radicle-design-system/Markdown.svelte';
-  import type { Application, Workstream } from '$lib/stores/workstreams/types';
+  import {
+    WorkstreamState,
+    ApplicationState,
+    type Application,
+    type Workstream
+  } from '$lib/stores/workstreams/types';
   import ActionRow from '$components/ActionRow.svelte';
 
   export let workstream: Workstream;
   export let applications: Application[] | undefined = undefined;
+
+  let activeApplication: Application | undefined = undefined;
+  let rejectedApplications: Application[] | undefined = undefined;
+
+  let applied: boolean =
+    $walletStore.connected &&
+    workstream.applicants?.includes($walletStore.address);
+  let creator: boolean =
+    $walletStore.connected && workstream.creator === $walletStore.address;
```

Comment:
Let's also check for `$connectedAndLoggedIn` here. Otherwise things might get confusing if you're showing up as logged out but your wallet is still connected.

Reply:
https://github.com/radicle-dev/workstreams-app/pull/45/commits/12e7a4a55c00ee8ee02c15c1c804e3aa0fa38101